source/uwp/Visualizer/AdaptiveCardVisualizer.csproj

```diff
@@ -11,8 +11,8 @@
     <AssemblyName>XamlCardVisualizer</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.14393.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.14393.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion>10.0.17134.0</TargetPlatformVersion>
```

Comment:
>17134 [](start = 32, length = 5)

Should this one also be 17763?

Reply:
yep :)

---
In reply to: [250401737](https://github.com/Microsoft/AdaptiveCards/pull/2322#discussion_r250401737) [](ancestors = 250401737)